megamek/src/megamek/common/PlanetaryConditions.java

```diff
@@ -683,14 +691,16 @@ public int getVisualRange(Entity en, boolean targetIlluminated) {
                 return 8;
             }
             return 2;
-        } else if ((weatherConditions == WE_HEAVY_SNOW)
-                || (weatherConditions == WE_HEAVY_RAIN)
-                || (weatherConditions == WE_DOWNPOUR)
+        } else if ((weatherConditions == WE_HEAVY_HAIL)
                 || (weatherConditions == WE_SLEET)
+                || (weatherConditions == WE_HEAVY_SNOW)
+                || (blowingSand && (windStrength >= WI_MOD_GALE))
+                || (lightConditions == L_FULL_MOON)
+                || (weatherConditions == WE_GUSTING_RAIN)
+                || (weatherConditions == WE_BLIZZARD)
+                || ((weatherConditions == WE_HEAVY_SNOW) && (windStrength >= WI_STRONG_GALE))
```

Comment:
why the wind strength check?

Reply:
it creates the blowing snow condition

Reply:
Isn't it not required because of the (weatherConditions == WE_HEAVY_SNOW) check above?

Reply:
So it is. Fixed.